examples/datadog-tracing/verify.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+export NAME=datadog-tracing
+export PORT_PROXY="${DATADOG_PORT_PROXY:-10000}"
```

Comment:
```suggestion
export PORT_PROXY="${DATADOG_PORT_PROXY:-10400}"
```
the reason for dynamically setting the port numbers is to prevent clashes if the compositions are run simultaneously